sympy/combinatorics/matrix_groups.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import print_function
+from sympy.core import Basic
+from math import sin, cos, pi, floor
+import random
+
+def MatrixMultiply(a, b):
+    """Returns the product of two matrices"""
+
+    _arr = [[] for i in range(len(a))]
+    for i in range(len(a)):
+        for j in range(len(b[0])):
+            c = 0
+            for k in range(len(b)):
+                c = c + (a[i][k] * b[k][j])
+            _arr[i].append(c)
+    return _arr
+
+def isPrime(p):
```

Comment:
This already exists in `sympy.ntheory.primetest` and it's called `isprime`. See http://docs.sympy.org/latest/modules/ntheory.html#sympy.ntheory.primetest.isprime